.github/workflows/admin.yaml

```
@@ -0,0 +1,35 @@
+name: Recurring Admin Tasks
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "55 * * * *"
+
+jobs:
+  publish-indices:
+    name: Publish Indices
+    runs-on: ubuntu-latest
+    steps:
+    - name: Publish english index
+      uses: fjogeleit/http-request-action@v1
```

Could we use the commit SHA instead of a version? For example for the [latest version](https://github.com/fjogeleit/http-request-action/commit/23ad54bcd1178fcff6a0d17538fa09de3a7f0a4d) it'd be this..:

```suggestion
      uses: fjogeleit/http-request-action@23ad54bcd1178fcff6a0d17538fa09de3a7f0a4d
```

A few months back there was a supply chain attack and one of our dependencies was affected too for another repo. (luckily no secrets leaked)

The malicious attackers rewrote versions, git SHAs are not rewritable unlike versions

---

I'm sorry I missed that comment before merging. We hadn't had any issues with versions *in* github actions so far, but if you prefer I can still change the version number to its commit id.